### PR TITLE
add Launch Excessively Capable Pod into k8saudit rule

### PR DESCRIPTION
**Pull request by hi120ki**
Signed-off-by: Hi120ki <12624257+hi120ki@users.noreply.github.com>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

/kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area plugins

> /area registry

> /area build

> /area documentation

**What this PR does / why we need it**:

In k8saudit, we can get pod's capabilities from `ka.req.pod.containers.add_capabilities` field.
This capabilities lead container escape and other attacks, so we need detect these capabilities are added.

And this rule leads false positive by privileged pod and ignore it by `falco_privileged_images` list.
In some case we have to add capabilities to pod, then I added `falco_capable_images` list (now this is empty) to add allow items.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by jasondellaluce** [comment]
cc @darryk10 @Kaizhe

File: plugins/k8saudit/rules/k8s_audit_rules.yaml [hunk 1]
```diff
@@ -741,3 +741,31 @@
...
+
+- list: falco_capable_images
+  items: []
+
+- rule: Launch Excessively Capable Pod
+  desc: >
+    Detect an attempt to start a pod with an excessively capable
+  condition: >
+    kevt and pod and kcreate and ka.req.pod.containers.add_capabilities intersects (excessively_capable_list)
+    and not ka.req.pod.containers.image.repository in (falco_privileged_images)
```
Comment (Kaizhe) [inline plugins/k8saudit/rules/k8s_audit_rules.yaml:766]: we need cover more exceptions here, `kube-proxy`, kube-dns` and any other CSI/CNI  services more or less use extra capabilities.

Comment (hi120ki) [reply plugins/k8saudit/rules/k8s_audit_rules.yaml:766]: I see. But I'm now testing only in GKE, and there are only `NET_BIND_SERVICE` `DAC_OVERRIDE` `SETGID` caps except privileged one.
Could you tell me or create suggestion about the pods we should ignore in other Kubernetes clusters like on AKS, EKS?

Comment (leogr) [reply plugins/k8saudit/rules/k8s_audit_rules.yaml:766]: cc @alacuku

**Comment by jasondellaluce** [comment]
/remove-lifecycle rotten

Any news on this? What is the state of review?

**Comment by AlbertoPellitteri** [comment]
Hello everyone!
Taking a look at this rule I think an error was made by reporting the capabilities of the `excessively_capable_list` list.
From the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-capabilities-for-a-container):  

> Note: Linux capability constants have the form CAP_XXX. But when you list capabilities in your container manifest, you must omit the CAP_ portion of the constant. For example, to add CAP_SYS_TIME, include SYS_TIME in your list of capabilities.

**Pull request closed without merging**